File: src/main/java/io/github/kingg22/api/vacunas/panama/service/IDoctorService.java
package io.github.kingg22.api.vacunas.panama.service;

import io.github.kingg22.api.vacunas.panama.persistence.entity.Doctor;
import java.util.Optional;
import java.util.UUID;

public interface IDoctorService {
    Optional<Doctor> getDoctorByUserID(UUID idUser);

    Optional<Doctor> getDoctorById(UUID idDoctor);
}

File: src/main/kotlin/io/github/kingg22/api/vacunas/panama/service/IDoctorService.kt
package io.github.kingg22.api.vacunas.panama.service

import io.github.kingg22.api.vacunas.panama.persistence.entity.Doctor
import java.util.Optional
import java.util.UUID

interface IDoctorService {
    fun getDoctorByUserID(idUser: UUID): Optional<Doctor>

    fun getDoctorById(idDoctor: UUID): Optional<Doctor>
}
